sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/DedicatedHostUpdate.cs

```diff
@@ -148,9 +148,12 @@ public DedicatedHostUpdate(IDictionary<string, string> tags = default(IDictionar
         /// </exception>
         public virtual void Validate()
         {
-            if (PlatformFaultDomain < 0)
+            if (PlatformFaultDomain != null)
```

Comment:
not sure why this file is being updated. does your swagger change have anything related to this?
if not, could you please take this change out of the PR